src/plugins/dashboard/public/dashboard_api/track_panel.ts

```diff
@@ -73,7 +73,7 @@ export function initializeTrackPanel(untilEmbeddableLoaded: (id: string) => Prom
           };
           return;
         }
-        panelRef.scrollIntoView({ block: 'nearest' });
+        panelRef.scrollIntoView({ block: 'start' });
```

Comment:
Was this an intentional change?

Reply:
Yeah this was necessary for fixing functional tests, and I think it's the desired behavior. When we set it to block: `start`, it will scroll the top of the panel to align with the top of the page.

Reply:
Ugh, I didn't refresh the page before commenting, so I didn't see that @cqliu1 already answered this. 🤦‍♂️